frontend/public/components/edit-yaml.jsx

```diff
@@ -324,6 +324,12 @@ export const EditYAML_ = connect(stateToProps)(
           return;
         }
 
+        // If this is NOT a namespaced resource and namespace is provided, display an error.
+        if (obj.metadata.namespace && !model.namespaced) {
+          this.handleError('"metadata.namespace" field found in YAML is not necessary.');
```

Comment:
I might make the message a little stronger since it's an error, something like

```
${obj.kind} is a cluster-scoped resource and cannot have a "metadata.namespace" field.